### ISSUE TITLE
Found an Emission Rate Slider on Screen 1

### ISSUE THREAD

**Issue by terracoda** [phetsims/quantum-wave-interference#71]
There is an emission rate slider on screen 1 between intensity and the experiment type radio group.
I don't think it is supposed to be there.

<img width="942" height="460" alt="Image" src="https://github.com/user-attachments/assets/5b238a3e-fd67-4e5b-8b77-723d27a7750c" />

**Comment by samreid** [comment]
There is an emission rate slider on the other scenes, but on the photon scene, I cannot find the Emission Rate slider in either the a11y view or VoiceOver. I'll likely reach out to others for help in reproducing the problem.

**Comment by terracoda** [comment]
I am only finding it when I am using VoiceOver, so I figured it was in the code somewhere, but not hidden properly.

**Comment by samreid** [comment]
@terracoda said:

> there is still something weird about the Source Intensity slider on Screen 1. Using arrow keys on screen 1, I stumble upon a source intensity slider 4 times. Once before the help text which is expected and then 3 times after the help text and before the the h4 heading for Particle Type which is not expected.

> There has got to be a bug in there somewhere.
> Sorry, the 3 sliders after are for the emission rate slider.
> Which are not supposed to be on screen 1.

**Comment by samreid** [comment]
I am able to access all 3 emission rate slides on screen 1, but only when the massive particles are showing. So I will schedule this for collaboration and we can investigate together.

**Comment by samreid** [comment]
Even though I was unable to reproduce the problem, I had an idea about how to fix it. @terracoda can you please test after that fix?

**Comment by terracoda** [comment]
I still hear it (3 times with cursor keys), but I do not see it anywhere on the console. And it is not focusable with the Tab key. I have no idea what is going on.

<img width="675" height="169" alt="Image" src="https://github.com/user-attachments/assets/d8796127-d89f-4110-9c3a-dee8d39b6609" />

**Comment by terracoda** [comment]
Oh, I found it. The code is using the HTML hidden attribute, but I think aria-hidden="true/false" might have better AT support. The aria-hidden attribute will remove an element from the accessibility tree. Maybe the HTML hidden attribute is less reliable in doing that. There is some complexity around all the ways to hide things, see https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-hidden

<img width="806" height="58" alt="Image" src="https://github.com/user-attachments/assets/feef5cd7-93e1-4222-9940-eb7c6f9b56eb" />

**Comment by terracoda** [comment]
I am going to ask @jessegreenberg if he has ever used HTML hidden in our API.

**Comment by terracoda** [comment]
@jessegreenberg said we use HTML hidden. And also VO has a know issue see also https://github.com/phetsims/center-and-variability/issues/644

But he is investigating because all three scenes are hidden, but I am only hear the Emission Rate slider - and I hear it 3 times, one for each scene. There's something about that slider that is being weird.

**Comment by terracoda** [comment]
I am still wondering if aria-hidden would work more consistently, but that's a bigger more general question.

**Comment by jessegreenberg** [comment]
I was not able to reproduce this problem on main. I tested with macOS Tahoe 26.3.1 with Safari.

I wouldn't expect `aria-hidden` would be better here. `hidden` should hide components both visually and in the accessibility tree while `aria-hidden` only hides in the accessibility tree, keeping it visually displayed. We could try setting both as a workaround.

@terracoda can you please share platform info?

EDIT: Also, @terracoda can you please share any settings you might be usign with VO?

**Comment by samreid** [comment]
I think we can work around this problem using ToggleNode instead of hidden. I did so in the commit. I tested with `?stringTest=dynamic` and it behaves nicely. Comparing to before the commit, I see the detector screen on experiment screen moved down a few pixels, but I measured in photoshop and it seems it was actually off centered before. 

I have a few more internal improvements to the implementation before I request a spot check from @matthew-blackman

**Comment by samreid** [comment]
OK I think it's working better now, @matthew-blackman can you please spot check and report any layout problems? Please test screen 1, in a few scenes, with a few settings of `?stringTest=dynamic` string widths.

**Comment by matthew-blackman** [comment]
Tested and found no issues with regard to the Emission Rate slider. Closing.

**Comment by samreid** [comment]
Layout broken on high intensity screen

**Comment by jessegreenberg** [comment]
Hey FWIW, it isn't clear to me why `ToggleNode` would help here. `hidden` is applied by scenery with changes to `visible` and `accessibleVisible` so I would think `ToggleNode` would behave the same. Only mentioning in case `ToggleNode` complicated the layout and you want to remove it.

**Comment by samreid** [comment]
@jessegreenberg ToggleNode supports this option:

```ts
  // Determine whether unselected children (the ones not displayed) are in the scene graph.
  // - If included (the default), unselected children are in the scene graph and hidden via setVisible(false). In this case
  //   the layout is the union of the bounds of all children (visible and invisible).
  // - If excluded, children are added to the scene graph when selected and removed when not selected. The ToggleNode has
  // the bounds of its selected child. This option can sometimes improve performance. Children added to the ToggleNode
  // outside the constructor will not be managed correctly.
  unselectedChildrenSceneGraphStrategy?: 'included' | 'excluded';
```

For our case, we went with a simpler implementation in `QuantumWaveInterferenceToggleNode` which also changes the `children`.

@matthew-blackman can you please verify the layout, and close if ok?

**Comment by terracoda** [comment]
I am not sure how the code was changed, but I do not hear the emission rate slider anymore in Safari or in Chrome when using MacOS VoiceOver Sequoia 15.6.1.

**Comment by terracoda** [comment]
@samreid, thanks for clarifying that the hidden content that my version of VO could still see was removed from the PDOM. This is a very good solution since older versions of VO have known bugs detecting hidden content.

I think this issue can be closed.

**Comment by jessegreenberg** [comment]
> @jessegreenberg ToggleNode supports this option:

I see, thank you for showing me!
